src/main/java/com/example/enterprise/service/ExpenseService.java

```diff
@@ -8,6 +8,11 @@
 
 import java.util.List;
 
+/**
+ * The ExpenseService class provides necessary logic for operation methods related to Expense
+ * such as save, list, and search. It implements all the methods from Expense service interface.
+ *
+ */
```

Comment:
Added Javadoc to the ExpenseService class as it is a good coding convention to follow. It provides the explanation to an outsider that what that class is trying to do. 
Here is a link to an article: https://www.tutorialspoint.com/java/java_documentation.htm